include/constants.h

```diff
@@ -87,6 +88,7 @@ namespace reinforcement_learning {  namespace name {
 namespace reinforcement_learning {  namespace value {
       const char *const AZURE_STORAGE_BLOB = "AZURE_STORAGE_BLOB";
       const char *const NO_MODEL_DATA = "NO_MODEL_DATA";
+      const char* const MODEL_HTTP_API = "MODEL_HTTP_API";
```

Comment:
This does not follow the naming conventions. Something closer to the following.

NO_MODEL_DATA
FILE_MODEL_DATA
HTTP_MODEL_DATA